engine/vcs/gitlab/oauth_consumer.go

```diff
@@ -119,7 +119,7 @@ func (g *gitlabConsumer) AuthorizeToken(ctx context.Context, state, code string)
 var instancesAuthorizedClient = map[string]*gitlabClient{}
 
 //GetAuthorized returns an authorized client
-func (g *gitlabConsumer) GetAuthorizedClient(ctx context.Context, accessToken, accessTokenSecret string) (sdk.VCSAuthorizedClient, error) {
+func (g *gitlabConsumer) GetAuthorizedClient(ctx context.Context, accessToken, accessTokenSecret string, _ int64) (sdk.VCSAuthorizedClient, error) {
```

Comment:
Maybe the name of the last parameter should be set, to help understanding its usage.